### DISCUSSION TITLE
Disable typing out snippet placeholder/template

### DISCUSSION THREAD

**Discussion by Formulacoder** [mark-wiemer/ahkpp#361] [Q&A] [answered]
Hi,

First of all, I'm quite new to coding and VS Code so i might not use the right terms. Hopefully you will understand me either way. If not please ask me to clarify!

I'm going mad because snippets are being inserted/typed out in the editor.
IE:
I type "msgb" and hit tab to auto complete. This i want to keep.
When tab is pressed this is what is typed into editor: MsgBox, [ Options, Title, Text, Timeout]
I don't want it to type out [ Options, Title, Text, Timeout]. It just ends up with me hitting ctrl+delete a couple of times to erase it.

I'm not even sure this extension is the problem but since i don't experience this with other languages i assume it is.
Maybe there is an easy way to remove the template after its been typed out, that would be fine as well.

The dream scenario is that msgb -> tab autocompletes it.
Then i can use ctrl+space to view the template. Like this in Python:
![image](https://github.com/mark-wiemer/ahkpp/assets/134759613/746f97b7-a6ed-41dd-b393-d17e07226bfe)

Please help me, I have googled for hours and investigated how this all works but i can't find a solution.

**Comment by mark-wiemer** [comment, marked as answer]
Hi, yes, my extension is the problem :/ this is a known issue and the best resolution currently is to, well, disable my extension :( I'm getting back into active development now, I'll see if there's an easy fix for this. Please review the open issues to see if one matches your concern. If you can't find one, you're welcome to open one yourself.
